pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -294,7 +294,17 @@ stdenv.mkDerivation (
       ++
         lib.optional (lib.versionAtLeast release_version "15")
           # Just like the `llvm-lit-cfg` patch, but for `polly`.
-          (getVersionFile "llvm/polly-lit-cfg-add-libs-to-dylib-path.patch");
+          (getVersionFile "llvm/polly-lit-cfg-add-libs-to-dylib-path.patch")
+      ++
+        lib.optional (lib.versions.major release_version == "20")
+          # https://github.com/llvm/llvm-project/pull/139822 adds a commit which didn't get backported but is necessary for tests.
+          (
+            fetchpatch {
+              url = "https://github.com/llvm/llvm-project/pull/139822/commits/340d19927571709999c006f13b648d3e632a79fa.patch";
+              stripLen = 1;
+              hash = "sha256-CZBTZKzi4cYkZhgTB5oXIo1UdEAArg9I4vR/m0upSRk=";
+            }
```

Comment:
This commit does not exist upstream and is liable to be garbage collected. It was merged as https://github.com/llvm/llvm-project/commit/ff2e8f93f6090965e82d799af43f6dfef52baa66.

Reply:
Done, didn't see this got merged heh.

Reply:
Gotcha, wasn't sure if it were fine or not because the patch was just a backport and I'm not sure how GH handles GC'ing commits from branches deleted but used in a PR.